crap-advisor-api/build.gradle.kts

```diff
@@ -17,48 +16,54 @@ kapt {
     }
 }
 
+noArg {
+    annotations(
+        "jakarta.persistence.Entity",
+        "jakarta.persistence.MappedSuperclass",
+        "jakarta.persistence.Embeddable"
```

Comment:
Explain this one.

Reply:
hibernate needs entities to have an empty default constructor, in java we can use lombok, in kotlin we can use this plugin that generates constructors automatically for classes marked with those annotations

dizzy technique man